src/backend/TdConsumer.ts

```diff
@@ -1,6 +1,6 @@
 import { Servient } from '@node-wot/core';
```

Comment:
adding a note somewhere in this file that coap has to be excluded until node-aead-crypto issue is resolved